.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
+    [
+      "env",
+      {
+        "targets": {
+          "browsers": ["last 2 versions", "last 1 ie version"]
+        }
+      }
+    ]
+  ],
+  "plugins": ["transform-async-to-generator"]
```

Comment:
Would you mind changing the legacy_app example to use async/await for demonstration purposes?